### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by buchi**
Allows to run opengever.core with Docker.

See https://github.com/4teamwork/opengever.core/blob/dockerize/docker/README.md for a detailed description.

Only a few changes are made in the application code:
- Disable logging into log file when running bundle import TTW.
- Disable ogds sync log file when running bundle import TTW.

## Checklist

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

JIRA: https://4teamwork.atlassian.net/browse/ROAD-1947

### CODE REVIEW

**Review by lukasgraf — APPROVED** [review]
LGTM! 👍 🎉 

I'm not a fan of the redundant versions in the`requirements.txt` vs. `versions.cfg` either. But I think we could add a test that makes sure that they're in sync, and eventually also a `zest.releaser` hook that automatically builds them. I'd consider that out of scope for this PR though.

File: docker-compose.yml [hunk 1]
```diff
@@ -153,3 +153,10 @@ services:
     profiles:
       - all
       - ianus
+
+networks:
+  default:
+    ipam:
+      driver: default
+      config:
+        - subnet: 172.16.4.0/24
```
[the remark this answers is not in the document]
Comment (buchi) [reply docker-compose.yml:162]: I'm not intending to do this in production. I suspect that the overlapping networks problem is just an issue with Docker for macOS.

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+### Import a bundle
+
+You can either create your own bundle or use the provided example.
+
+Got to http://localhost:8080/ogsite/@@import-bundle and provide the .json files
+from your bundle. E.g. `configuration.json` and `workspaceroots.json` from
+`example_bundles/teamraum.oggbundle`.
+
+After importing the bundle you should now be able to access the application
+using the hostname you defined at port 8088.
```
[the remark this answers is not in the document]
Comment (buchi) [reply docker/README.md]: amended

File: docker/README.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+### Disable Secure Session Cookie
+
+Currently we don't have an HTTPS setup yet. Thus Plone's session cookie secure
+flag needs to be disabled. Got to the properties of the PAS session plugin
+(acl_users/session) an uncheck the "Only Send Cookie Over HTTPS" setting.
+
+
+## Building the Docker images
+
+To build all images simply run:
```
[the remark this answers is not in the document]
Comment (buchi) [reply docker/README.md:128]: amended

**Pull request merged**